man/man8/zpool.8

```diff
@@ -579,6 +579,27 @@ Keep in mind is that the \fBashift\fR is \fIvdev\fR specific and is not a \fIpoo
 
 .sp
 .LP
+The following properties can be set at creation time:
+.ne 2
+.na
+.sp
+\fB\fBsegregate_log\fR=\fIon\fR\fR
+.sp
+\fB\fBsegregate_metadata\fR=\fIon\fR\fR
+.sp
+\fB\fBsegregate_smallblks\fR=\fIon\fR\fR
+.ad
+.sp .6
+.RS 4n
+The segregate properties direct the allocating vdevs in the pool
+to segregate a portion of their metaslabs to the allocation class
```

Comment:
`metaslabs` isn't a user-visible concept.  Let's just say "a portion of their space to the allocation of specific types of blocks".